.gitignore

```diff
@@ -7,3 +7,4 @@ build/
 packages
 .packages
 pubspec.lock
+*.iml
```

Comment:
Tip: if you put files generated by your editor in [your global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore), you don't need to add them separately to all the repos you work on.